toolkit/committee-selection/selection/src/ariadne_v2.rs

```diff
@@ -217,9 +217,9 @@ mod tests {
 			r2_count += committee.iter().filter(|c| **c == "R2").count();
 		}
 		let tolerance = 5;
-		assert!(1333 - tolerance <= p1_count && p1_count <= 1333 + tolerance);
-		assert!(1200 - tolerance <= r1_count && r1_count <= 1200 + tolerance);
-		assert!(1800 - tolerance <= r2_count && r2_count <= 1800 + tolerance);
+		assert!(1343 - tolerance <= p1_count && p1_count <= 1343 + tolerance);
+		assert!(1220 - tolerance <= r1_count && r1_count <= 1220 + tolerance);
+		assert!(1780 - tolerance <= r2_count && r2_count <= 1780 + tolerance);
```

Comment:
same here, I put the real numbers temporary to show how values changed

Reply:
Let me investigate it. The tolerance was `5`, now it seems to be more like `25`. Anyway, the number were expected values. Expected values don't change. Please read the test.

Reply:
I've run the test for 100 000 iterations and results are:
```
New rand: 133415, 120136, 179864 
Old rand: 133465, 120145, 179855
```
So, I think we are good to go with this updated rand crates.